theme/template.sublime-theme

```diff
@@ -0,0 +1,1026 @@
+		},
+
+		// File Icons
+		{
+		  "class": "icon_file_type",
+		  "layer0.tint": "var(muted)",
+		  "layer0.opacity": 1,
+		  "content_margin": [8, 8]
+		},
+		// File Icons: Hovered
+		{
+		  "class": "icon_file_type",
+		  "parents": [{"class": "tree_row", "attributes": ["hover"]}],
+		  "layer0.tint": "var(muted)",
+		  "layer0.opacity": 1
+		},
+		// File Icons: Selected
+		{
+		  "class": "icon_file_type",
+		  "parents": [{"class": "tree_row", "attributes": ["selected"]}],
+		  "layer0.tint": "var(text)",
+		  "layer0.opacity": 1
+		},
+
+		// Tabs
+		{
+			"class": "tabset_control",
+			"layer0.texture": "",
+			"layer0.opacity": 1.0,
+			"layer0.tint": "var(surface)",
+			"layer0.inner_margin": 0,
+			"content_margin": [0, 4, 0, 4],
+			"tab_overlap": 1,
+			"tab_height": 26,
+			"connector_height": 0,
+			"spacing": 5,
+		},
+		{
+			"class": "tabset_control",
+			"settings": ["themed_title_bar"],
+			"content_margin": [0, 0, 0, 4],
+		},
+
+		{
+			"class": "tab_connector",
+			"layer0.texture": "",
+			"layer0.opacity": 1.0,
+			"tint_index": 0,
+		},
+
+		{
+			"class": "tab_control",
+			"layer0.texture": "",
+			"layer0.opacity": 1.0,
+			"layer0.tint": "var(surface)",
+			"layer0.inner_margin": 0,
+			"layer1.texture": "Theme - Default/common/button.png",
+			"layer1.opacity": 1.0,
+			"layer1.tint": "var(overlay)",
+			"layer1.inner_margin": 4,
+			"layer2.texture": "Theme - Rose Pine/images/tab_square_highlight.png",
+			"layer2.opacity": 0.0,
+			"layer2.inner_margin": 2,
+			"layer3.texture": "Theme - Default/common/tab_square_divider.png",
+			"layer3.inner_margin": [2, 0, 2, 0],
+			"layer3.opacity": "var(transparent)",
+			"layer3.tint": "var(highlight-med)",
+			"tint_index": 4,
+			"tint_modifier": "transparent",
+			"accent_tint_index": 2,
+			"content_margin": [12, 3, 3, 4],
+			"hit_test_level": 0.3
+		},
+
+		// We don't use animation for the selected tab opacity changed due
+		// to the hover state syncing with the hover state of the
+		// sheet_contents, which has the background_modifier that can not
+		// be animated
+		{
+			"class": "tab_control",
+			"attributes": ["selected"],
+			"layer1.opacity": 1.0,
+		},
+		{
+			"class": "tab_control",
+			"attributes": ["!selected"],
+			"layer1.opacity": "var(transparent)",
+		},
+		{
+			"class": "tab_control",
+			"attributes": ["!selected", "hover"],
+			"layer1.opacity": "var(transparent)",
+		},
+		// Tabs - modified highlight
+		{
+			"class": "tab_control",
+			"attributes": ["dirty"],
+			"settings": {
+				"highlight_modified_tabs": true
+			},
+			"layer2.opacity": 0.8
+		},
+		{
+			"class": "tab_control",
+			"attributes": ["dirty", "selected"],
+			"settings": {
+				"highlight_modified_tabs": true
+			},
+			"layer2.opacity": 1.0
+		},
+		{
+			"class": "tab_control",
+			"attributes": ["dirty", "!selected"],
+			"settings": {
+				"highlight_modified_tabs": true
+			},
+			"layer2.texture": "Theme - Rose Pine/images/tab_square_pinstripe.png",
```

Comment:
Should this be Rosé instead of Rose? If it works then ignore this suggestion.

```suggestion
			"layer2.texture": "Theme - Rosé Pine/images/tab_square_pinstripe.png",
```